pyccel/codegen/printing/pycode.py

```diff
@@ -175,6 +175,14 @@ def _print_NativeComplex(self, expr):
     def _print_Variable(self, expr):
         return self._print(expr.name)
 
+    def _print_DottedVariable(self, expr):
+        rhs_code = self._print_Variable(expr)
+        lhs_code = self._print(expr.lhs)
+        return f"{lhs_code}.{rhs_code}"
+
+    def _print_DottedFunctionCall(self, expr):
+        return (f"{self._print(expr.func_name)}({', '.join(self._print(arg) for arg in expr.args)})\n")
```

Comment:
I think the readability of this line could be improved by introducing intermediate variables. Ie:
```suggestion
        func = self._print(expr.func_name)
        args = ', '.join(self._print(arg) for arg in expr.args)
        return (f"{func}({args})\n")
```